rocketpy/simulation/flight.py

```diff
@@ -1422,6 +1429,30 @@ def u_dot(self, t, u, post_processing=False):
             drag_coeff = self.rocket.power_off_drag.get_value_opt(free_stream_mach)
         rho = self.env.density.get_value_opt(z)
         R3 = -0.5 * rho * (free_stream_speed**2) * self.rocket.area * (drag_coeff)
+        for airbrakes in self.rocket.airbrakes:
```

Comment:
Just out of curiosity...
Do you see a case where someone would need to add multiple air brakes to a rocket?

No changes requested, just for personal knowledge.

Reply:
No, but I guess it is possible .... ?